purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/CustomerCenterConfigData.kt

```diff
@@ -0,0 +1,260 @@
+            APPLE_SUBSCRIPTION_MANAGE,
+
+            @SerialName("google_subscription_manage")
+            GOOGLE_SUBSCRIPTION_MANAGE,
+
+            @SerialName("amazon_subscription_manage")
+            AMAZON_SUBSCRIPTION_MANAGE,
+
+            @SerialName("platform_mismatch")
+            PLATFORM_MISMATCH,
+            ;
+
+            val defaultValue: String
+                get() = when (this) {
+                    NO_THANKS -> "No, thanks"
+                    NO_SUBSCRIPTIONS_FOUND -> "No Subscriptions found"
+                    TRY_CHECK_RESTORE -> "We can try checking your Apple account for any previous purchases"
+                    RESTORE_PURCHASES -> "Restore purchases"
+                    CANCEL -> "Cancel"
+                    BILLING_CYCLE -> "Billing cycle"
+                    CURRENT_PRICE -> "Current price"
+                    EXPIRED -> "Expired"
+                    EXPIRES -> "Expires"
+                    NEXT_BILLING_DATE -> "Next billing date"
+                    REFUND_CANCELED -> "Refund canceled"
+                    REFUND_ERROR_GENERIC -> "An error occurred while processing the refund request. Please try again."
+                    REFUND_GRANTED -> "Refund granted successfully!"
+                    REFUND_STATUS -> "Refund status"
+                    SUB_EARLIEST_EXPIRATION -> "This is your subscription with the earliest expiration date."
+                    SUB_EARLIEST_RENEWAL -> "This is your subscription with the earliest billing date."
+                    SUB_EXPIRED -> "This subscription has expired."
+                    CONTACT_SUPPORT -> "Contact support"
+                    DEFAULT_BODY -> "Please describe your issue or question."
+                    DEFAULT_SUBJECT -> "Support Request"
+                    DISMISS -> "Dismiss"
+                    UPDATE_WARNING_TITLE -> "Update available"
+                    UPDATE_WARNING_DESCRIPTION ->
+                        "Downloading the latest version of the app may help solve the problem."
+                    UPDATE_WARNING_UPDATE -> "Update"
+                    UPDATE_WARNING_IGNORE -> "Continue"
+                    PLATFORM_MISMATCH -> "Platform mismatch"
+                    PLEASE_CONTACT_SUPPORT_TO_MANAGE -> "Please contact support to manage your subscription."
+                    APPLE_SUBSCRIPTION_MANAGE ->
+                        "You can manage your subscription by using the App Store app on an Apple device."
+                    GOOGLE_SUBSCRIPTION_MANAGE ->
+                        "You can manage your subscription by using the Play Store app on an Android device"
+                    AMAZON_SUBSCRIPTION_MANAGE ->
+                        "You can manage your subscription in the Amazon Appstore app on an Amazon device."
+                }
+        }
+
+        fun commonLocalizedString(key: CommonLocalizedString): String {
+            return localizedStrings[key.name.lowercase()] ?: key.defaultValue
```

Comment:
I didn't fully like this... It means the enum name must always match the key in uppercase. Want to write a test to make sure this is setup correctly.

Reply:
Test written in https://github.com/RevenueCat/purchases-android/pull/1859/commits/8cb3f5f814524f0a1e0f8b5b633be516005476dc

Reply:
In this case as well I wouldn't be opposed to having the enum just match the JSON key. (Testing would still be good, but the "rule" is simpler.)